device_coupler/heartbeat_scheduler.py

```diff
@@ -0,0 +1,39 @@
+"""Schedule heart beat with functions that have to be called in order"""
+from __future__ import absolute_import
+import threading
+
+from device_coupler.utils import get_logger
+
+
+class HeartbeatScheduler:
+    """Heart beat scheduler"""
```

Comment:
Heartbeat is one word (check comments here and above)

Reply:
Changed